pynamodb/attributes.py

```diff
@@ -525,6 +536,53 @@ def deserialize(self, value):
         return json.loads(value)
 
 
+class TTLAttribute(Attribute):
+    """
+    An attribute that signifies when how long this item can live for.
+    This can take in an int, timedelta, or datetime.  If a timedelta is passed in,
+    we assume the expiration time is datetime.utcnow() + timedelta.
+
+    It is recommended to pass in a timedelta.
+    """
+    attr_type = NUMBER
+
+    def __set__(self, instance, value):
+        """
+        Force a TTLAttribute to have a UTC datetime value
+        """
+        if not instance or self._is_map_attribute_class_object(instance):
+            return
+        if isinstance(value, timedelta):
+            value = int(time.time() + value.total_seconds())
```

Comment:
`time.time()` is a bug as it's timezone-dependent. Please fix and add tests to catch this case

Reply:
How is it timezone dependent? I've never heard about that  -  https://docs.python.org/2/library/time.html#time.time

Reply:
@garrettheel `time.time()` is seconds since epoch and the epoch is unambiguously defined in GMT

Reply:
Yep, my bad here. I misunderstood the docs, sorry for the misdirection.